%YAML:1.0
---
img_w: 848
img_h: 800
new_w: 600
new_h: 400
fov_v: 60
K1: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [830.0567541288623, 0., 426.41405242269826, 
          0., 828.6042739331378, 385.8680729738739, 
          0., 0., 1.]
K2: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [824.0953840443088, 0., 427.70974831604207, 
          0., 822.4635172227553, 397.0865445739266, 
          0., 0., 1.]
D1: !!opencv-matrix
   rows: 1
   cols: 4
   dt: d
   data: [ 0.18687417637223752, -1.130421637467151, -7.38476106514311e-05, 0.0005695787723648452 ]
D2: !!opencv-matrix
   rows: 1
   cols: 4
   dt: d
   data: [ 0.1669876453839371, -1.0239112986785401, 0.0017611098715461424, 4.816356547093389e-05 ]
xi1: !!opencv-matrix
   rows: 1
   cols: 1
   dt: d
   data: [ 1.9046829074310034 ]
xi2: !!opencv-matrix
   rows: 1
   cols: 1
   dt: d
   data: [ 1.8779872465806613 ]
R: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [0.9999915218248502, -0.0023193622763987066, -0.0034024751354170516, 
          0.00230503363924067, 0.999988484941289, -0.0042091334908216085, 
          0.003412198461151049, 0.004201254985406837, 0.9999853530718364]
T: [-0.06360918496694491, -0.0012407983766202222, -0.000609637685332673]
